### ISSUE TITLE
ui: Darken text to increase contrast ratio for visually impaired users

### ISSUE THREAD

**Issue by rbreeze** [argoproj/argo-cd#5408]
# Summary

The [WCAG 2.0](https://www.w3.org/TR/WCAG20/) recommend a minimum contrast ratio of 4.5:1 (text:background) for its AA standard, and a 7:1 ratio for AAA. The current contrast ratio of `$argo-color-gray-6` (used for most text in the UI) to a white background is [4.15:1](https://webaim.org/resources/contrastchecker/?fcolor=6D7F8B&bcolor=FFFFFF), which while only slightly below the AA standard, falls quite short of the AAA criteria. 

# Proposal

Simply switching from `$argo-color-gray-6` to `$argo-color-gray-7`for text on white backgrounds nets a contrast ratio of [7.42:1](https://webaim.org/resources/contrastchecker/?fcolor=495763&bcolor=FFFFFF). 

Before:
<img width="243" alt="Screen Shot 2021-02-04 at 9 52 58 AM" src="https://user-images.githubusercontent.com/6250584/106934264-c3d8e600-66ce-11eb-85de-45f789c7504c.png">

After:
<img width="250" alt="Screen Shot 2021-02-04 at 9 53 22 AM" src="https://user-images.githubusercontent.com/6250584/106934307-d2270200-66ce-11eb-96f3-0071453addfa.png">

**Comment by reginapizza** [comment]
Hi @rbreeze, I think that Argo CD UI in many places does not meet WebAIM contrast standards (for example, [this is also another issue with that](https://github.com/argoproj/argo-cd/issues/5260)). I plan on bringing up this as an issue at the UI/UX-focused ArgoCD Contributors call this coming Thursday :)

**Comment by rbreeze** [comment]
I agree, there are a lot of accessibility improvements that can be made. #5458 starts work on this one
